scapy/data.py

```diff
@@ -310,11 +311,36 @@ def _process_data(fdesc):
     return dct
 
 
+class EtherDA(DADict):
```

Comment:
Why do you use a specific subclass for that?
One may use in their own script another `DADict` "the old way", and I think they would deserve a proper warning too.
Moreover, this would simplify the code as it would remove the need for an extra parameter (`_cls`) in `load_protocols()`.
What do you think?

Reply:
`conf.mib` (in `scapy/asn1/mib.py`) is a `Dict[str, str]`. This funtion wouldn't work in this case and would wronly revert the keys/values, so it can't be put back in the DADict without adding some extra logic (specifying the value type, the key type....) which i find unnecessary.